common/network_constants.cc

```diff
@@ -53,7 +53,6 @@ const char kTwitterRedirectURL[] = "https://mobile.twitter.com/i/nojs_router*";
 const char kCookieHeader[] = "Cookie";
 // Intentional misspelling on referrer to match HTTP spec
 const char kRefererHeader[] = "Referer";
-const char kUserAgentHeader[] = "User-Agent";
 const char kBravePartnerHeader[] = "X-Brave-Partner";
```

Comment:
Can this be deleted?

Reply:
I think so. this is only used by UA whitelist.

Reply:
Sorry, just to be clear I'm talking about `kBravePartnerHeader`.

Reply:
I think it also can be deleted because it is only used by `OnBeforeStartTransaction_ReferralsWork()`.